tests/deployment/native_networks.py

```diff
@@ -64,12 +65,16 @@ def create_networks():
             # Assert that n0 and n1 are not on any network
             port_networks = self.get_port_networks(ports)
 
-            assert self.get_network(nodes[0].nics[0].port, port_networks) == set()
-            assert self.get_network(nodes[1].nics[0].port, port_networks) == set()
+            assert self.get_network(nodes[0].nics[0].port, port_networks) == set()  # noqa
+            assert self.get_network(nodes[1].nics[0].port, port_networks) == set()  # noqa
```

Comment:
I'd prefer to leave `# noqa` for places where there's really no reasonable way to comply with the guide.  If you can't come up with anything better, putting the `set()` on the next line like we're doing below seems acceptable.